mailpoet/lib/Config/Menu.php

```diff
@@ -502,6 +504,16 @@ private function registerAutomationMenu(bool $showEntries) {
       [$this, 'automationEditor']
     );
 
+    // Automation analytics
+    $this->wp->addSubmenuPage(
+      true,
```

Comment:
When you rebase on trunk, please use `self::AUTOMATIONS_PAGE_SLUG`. The JS method is no longer used since https://github.com/mailpoet/mailpoet/pull/4933/files.

Reply:
addressed in a183e254f